2021q3/copy_staging.adoc

```diff
@@ -0,0 +1,76 @@
+== Changes to UEFI boot on amd64 staging 
```

Comment:
I would drop "staging" here, it refers to an aspect of the boot but reads as if this is part of an "amd64 staging" branch.

How about simply "Improved amd64 UEFI boot"?